### DISCUSSION TITLE
nerdctl support?

### DISCUSSION THREAD

**Discussion by kbrierly** [mag37/dockcheck#42] [Ideas]
Do you have any plans to make dockcheck compatible with nerdctl? I'm looking to move away from docker to nerdctl and am looking for an application to at some level replace watchtower.

One difference I see is that the Image name selector is different. I did a quick replace in the script and it seemed to work.
docker => {{.Config.Image}}
nerdctl => {{.Image}}

**Comment by kbrierly** [comment]
After further use there seems to be too much missing inspect data for things like nerdctl compose.
So please disregard unless you have an interest.

Thanks!

**Comment by mag37** [comment]
Seems interesting! I don't have the time to dive down yet another container tool right now, but it looks promising.

If you'd come back to it later and stumble on anything that could make it work, I'll be happy to look into how to implement things.
